packages/widget/src/context/config.ts

```diff
@@ -3,14 +3,15 @@ import { createContext, ContextProvider } from '@lit/context';
 import type { WalletConnectOptions } from '@web3-onboard/walletconnect/dist/types';
 import type { HTMLTemplateResult, PropertyValues } from 'lit';
 import { html } from 'lit';
-import type { AppMetadata } from '@web3-onboard/common';
+import type { AppMetadata, WalletInit } from '@web3-onboard/common';
 import { BaseComponent } from '../components/common/base-component';
 import type { Theme } from '../interfaces';
 
 export interface ConfigContext {
   theme?: Theme;
   walletConnectOptions?: WalletConnectOptions;
   appMetaData?: AppMetadata;
+  walletSelectorWalletConfigurations?: WalletInit[];
```

Comment:
i know you just applied @mpetrunic's suggestion, so sorry for this, but still seems a bit weird, especially having word 'wallet' appear 2 times 🤣 
how about `walletConfigModules` or just `walletModules`? i think this way it's pretty self-explanatory and a lot shorter than current one :)

Reply:
@sztok7 yeah that is a difficult name for an attribute i agree 😅

Reply:
how about `walletInitModules` ?

Reply:
I'm fine with walletModules